modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -341,6 +342,24 @@ export class Tile3D {
     this._selectedFrame = 0;
   }
 
+  /**
+   * Memory usage of tile on GPU
+   */
+  _getGpuMemoryUsageInBytes(): number {
+    switch (this.type) {
+      case TILE_TYPE.MESH:
+      case TILE_TYPE.POINTCLOUD:
+        return this.content.byteLength;
+
+      case TILE_TYPE.SCENEGRAPH:
+        return getMemoryUsageGLTF(this.content.gltf);
```

Comment:
I suggest to move `gpuMemoryUsage` calculation to `3d-tiles` module? `gltf` dependency is not good as `tile` module is abstract by design. Feel free to set new property like `content.gpuMemoryUsageInBytes` as `content.byteLength` is not correct